agent/Makefile

```diff
@@ -53,6 +53,18 @@ util-scripts = \
 	require-rpm \
 	tool-meister
 
+# Scripts based on the Python Click package, which are generated during
+# installation.
+click-scripts = \
+	pbench-generate-token \
```

Comment:
Ha; they're all in alphabetical order except yours. ;-)

Reply:
Dang it!

Reply:
We can fix this is another commit.